game/sfx/game_theme.mp3.import

```diff
@@ -0,0 +1,19 @@
+[remap]
+
+importer="mp3"
+type="AudioStreamMP3"
+uid="uid://lvnjsdkmeyvi"
+path="res://.godot/imported/game_theme.mp3-9e64765ef2cdd95e45246d5b0d11f898.mp3str"
+
+[deps]
+
+source_file="res://game/sfx/game_theme.mp3"
+dest_files=["res://.godot/imported/game_theme.mp3-9e64765ef2cdd95e45246d5b0d11f898.mp3str"]
+
+[params]
+
+loop=false
```

Comment:
this probably should be set to true

Reply:
the player is set to loop, the file is not